packages/sommelier.rb

```diff
@@ -356,9 +422,9 @@ def self.install
     FileUtils.mkdir_p %W[#{CREW_DEST_PREFIX}/bin #{CREW_DEST_PREFIX}/sbin #{CREW_DEST_PREFIX}/etc/bash.d
                          #{CREW_DEST_PREFIX}/etc/env.d CREW_DEST_HOME]
     Dir.chdir('vm_tools/sommelier') do
-      system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+      system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install" unless ARCH == 'i686'
```

Comment:
Just abort at the start for i686?

Reply:
I'm leaving the startsommelier script, so we can still get some variables set inside containers, which is why I didn't set this all to return immediately during the build.